src/app/components/drawer/DrawerCustomLists.js

```diff
@@ -0,0 +1,180 @@
+                  isActive={activeItem.type === routePrefix && activeItem.id === search.dbid}
+                  key={search.id}
+                  routePrefix={routePrefix}
+                  savedSearch={search}
+                  teamSlug={team.slug}
+                  tooltip={search.title}
+                />
+              ))}
+            </>
+          }
+        </Collapse>
+      </React.Fragment>
+
+      {/* Dialog to create list */}
+
+      <NewSavedSearch
+        buttonLabel={<FormattedMessage defaultMessage="Create List" description="Label for a button to create a new list displayed on the left sidebar." id="drawerCustomLists.createList" />}
+        errorMessage={<FormattedMessage defaultMessage="Could not create list, please try again" description="Error message when creating new list fails" id="drawerCustomLists.newListErrorMessage" />}
+        listType={listType}
+        open={showNewListDialog}
+        routePrefix={routePrefix}
+        successMessage={<FormattedMessage defaultMessage="Filtered List created successfully" description="Success message when new list is created" id="drawerCustomLists.newListSuccessMessage" />}
+        team={team}
+        title={<FormattedMessage defaultMessage="New Custom Filtered List" description="Title for a dialog to create a new list displayed on the left sidebar." id="drawerCustomLists.newList" />}
+        onClose={() => { setShowNewListDialog(false); }}
+      />
+    </React.Fragment>
+  );
+};
+
+DrawerCustomListsComponent.propTypes = {
+  listType: PropTypes.string.isRequired,
+  location: PropTypes.string.isRequired,
+  routePrefix: PropTypes.string.isRequired,
+  savedSearch: PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+    filters: PropTypes.string.isRequired,
+  }).isRequired,
+  team: PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    dbid: PropTypes.number.isRequired,
+    slug: PropTypes.string.isRequired,
+    permissions: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const DrawerCustomLists = ({ listType, routePrefix, teamSlug }) => (
+  <QueryRenderer
+    environment={Relay.Store}
+    query={graphql`
+      query DrawerCustomListsQuery($teamSlug: String!, $listType: String!) {
+        team(slug: $teamSlug) {
+          dbid
+          slug
+          permissions
+          saved_searches(first: 10000, list_type: $listType) {
+            edges {
+              node {
+                id
+                dbid
+                title
+                is_part_of_feeds
+                ...SavedSearchesListItem_savedSearch
+              }
+            }
+          }
+        }
+      }
+    `}
+    render={({ error, props }) => {
+      if (!props || error) return null;
+
+      const { location } = window;
+
+      return (
+        <DrawerCustomListsComponent
+          listType={listType}
+          location={location}
```

Comment:
Does location need to be passed as a prop, since it's in the global `window`?

Reply:
Removed `location` from prop